From cdf486d8a7ff8745c7fa84dc4b79bb9506ec3498 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Mon, 28 Jan 2019 13:54:13 -0800
Subject: [PATCH 04/10] Fix install of public headers within subdirectories

There are several more public headers to install that live within
subdirectories, not at toplevel.
---
 inference-engine/src/inference_engine/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/inference-engine/src/inference_engine/CMakeLists.txt b/inference-engine/src/inference_engine/CMakeLists.txt
index 4559789..c71408e 100644
--- a/inference-engine/src/inference_engine/CMakeLists.txt
+++ b/inference-engine/src/inference_engine/CMakeLists.txt
@@ -127,7 +127,7 @@ configure_file(
 set(ConfigPackageLocation lib64/cmake/InferenceEngine)
 install(TARGETS ${TARGET_NAME} EXPORT ${TARGET_NAME} LIBRARY DESTINATION lib64)
 install(EXPORT ${TARGET_NAME} DESTINATION ${ConfigPackageLocation} NAMESPACE IE:: FILE targets.cmake)
-install(FILES ${PUBLIC_HEADERS} DESTINATION include/${TARGET_NAME})
+install(DIRECTORY ${PUBLIC_HEADERS_DIR}/ DESTINATION include/${TARGET_NAME} FILES_MATCHING PATTERN "*.h" PATTERN "*.hpp")
 install(FILES
     "${CMAKE_BINARY_DIR}/InferenceEngineConfig.cmake"
     "${CMAKE_BINARY_DIR}/InferenceEngineConfig-version.cmake"
-- 
2.20.1

